.github/workflows/update-nimbus-experiments.yml

```diff
@@ -0,0 +1,36 @@
+name: "Update Nimbus Experiments"
+
+on:
+  schedule:
+    - cron: '*/30 * * * *'
+
+jobs:
+  update-nimbus-experiments:
+    name: "Update Nimbus Experiments"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checkout Main Branch"
+        uses: actions/checkout@v2
+        with:
+          path: firefox-ios
+          ref: main
+      - name: "Update Experiments JSON"
+        id: update-experiments-json
+        uses: jeddai/update-initial-experiments@v1
```

Comment:
nit: This is something that you are planning to support for long term? 

I am asking as this lies in your repo and is not part of Mozilla's group of repos

Reply:
I would much prefer it be in the Mozilla-mobile org, is there any guidance for how to start that process/discussion?

Reply:
@jeddai I am not sure who to ask but here is our Mozilla Github guide that might be able to guide you

https://wiki.mozilla.org/GitHub

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1784126

Reply:
It's been moved and the workflow has been updated! https://github.com/mozilla-mobile/update-initial-experiments